### ISSUE TITLE
Implement the Art Item page with endpoint integrations

### ISSUE THREAD

**Issue by canatakan** [bounswe/bounswe2022group7#331]
As discussed in the [Mobile Team Meeting 3](https://github.com/bounswe/bounswe2022group7/wiki/MOBILE-Meeting-Notes-%233), I was assigned to prepare the Art Item page and complete the integration of those with the relevant endpoints. 

* Since our documentation is not ready yet, the Back-end Team has created a Wiki page containing the endpoints that are ready to be integrated. See: [Endpoint Table for Backend Requests](https://github.com/bounswe/bounswe2022group7/wiki/Endpoints).

* Some sub-tasks of this issue are as follows:
  - [x] Branch out from the `develop` branch
  - [ ] ~Connect the route from the homepage to the relevant Art Item pages~
  - [x] Design the Art Item page UI
  - [x] #339 (responsible person: @atillaturkmen)
  - [x] Create a pull request to merge the new branch `dev/android/art-item-page` into `develop`

**Deadline: 29/10/2022, 23:59**
**Reviewer: @musasimsekdotdll**

**Comment by canatakan** [comment]
- The task of integrating the network layer of Art Items will be a separate task and assigned to @atillaturkmen.
- He will complete the integration of the Art Item related endpoint: `art_item/{id}`.

**Comment by canatakan** [comment]
- The branch named `dev/android/art-item-page` has just been created from the `develop` branch.
- The initial development in this branch can be viewed in commit 7f1be8468ec88a6d166df4c5cbdf8f279bca04c9.
- In the mentioned commit, I added a very basic Art Item Page so we both can start to work on the same branch in parallel.

**Comment by canatakan** [comment]
The `ArtItem` and `ArtItemInfo` models have just been implemented according to the back-end team's feedback. More detailed information can be found at the commit 38afb84143fb4c773f2270269999baeedcdd9dc9.

**Comment by atillaturkmen** [comment]
I have implemented the connection of art item endpoint. You can find the relevant work in this commit: https://github.com/bounswe/bounswe2022group7/commit/be3345b6b6148bd7e4e34d6115d774c7902d5d73

**Comment by canatakan** [comment]
- The pull request #365 is now open and the final version of the Art Item related developments should be included.
- The task of *"Connecting the route from the homepage to the relevant Art Item pages"* is not completed since it is more about the Home Page integration. 
- Because of how urgent this task is, I am now assigning @atillaturkmen as the reviewer.

Please review the pull request mentioned above and act accordingly in your earliest convenience, so that we can complete the development as soon as possible.

**Comment by canatakan** [comment]
Also, a screenshot from the UI can be seen below for reference:

<img width="325" alt="Screen Shot 2022-11-01 at 00 34 31" src="https://user-images.githubusercontent.com/56476673/199116189-07269658-c86a-4476-89ab-4728a88428d2.png">

**Comment by atillaturkmen** [comment]
I reviewed the PR https://github.com/bounswe/bounswe2022group7/pull/365 and merged it. Therefore I am closing this issue.
